doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -23,6 +23,111 @@ Distribution
 
 The |ISE| is provided as a precompiled binary, which is part of the nRF54H20 SoC bundle and is provided independently from the |NCS| release cycle.
 
+Installing
+**********
+
+For new setups, follow the instructions found in :ref:`ug_nrf54h20_gs`.
+
+Updating IronSide SE
+********************
+
+Brief instruction
+=================
+
+To update the IronSide SE firmware, use the ``west ncs-ironside-se-update`` command with the following syntax:
+
+.. code-block:: console
+
+   west ncs-ironside-se-update --zip <path_to_soc_binaries.zip> --allow-erase
+
+The command accepts the following main options:
+
+* ``--zip``: Path to the IronSide SE release ZIP file (required)
+* ``--allow-erase``: Allow erasing the device during update (currently required)
+* ``--serial``: Specify the serial number of the target device
+* ``--firmware-slot``: Update only a specific firmware slot (``uslot`` for IronSide SE or ``rslot`` for IronSide SE Recovery)
+* ``--wait-time``: Timeout in seconds to wait for the device to boot (default: 2.0)
+
+Detailed update process
```

Comment:
We should make it more clear somehow that there are two ways to do it and that the user can stop reading
at this point if he is going with the easy option.

Reply:
We should also recommend updating both uslot and rslot I believe.